.github/workflows/tests.yml

```diff
@@ -23,5 +23,10 @@ jobs:
       - name: Run tests
         run: dart test
 
+      - name: Static code analysis
```

Comment:
What is this? it looks to be a dart formatting, which is cool but it's not static code analysis.

Reply:
> @ricardobrg what is this example/ stuff?

Pub is requiring an example app to improve our grade. I have added an empty example app, that will use the same code we have in the docs examples.

Reply:
> What is this? it looks to be a dart formatting, which is cool but it's not static code analysis.

Dart pub is asking for Dart format and calling it static analysis:
<img width="815" alt="Captura de Tela 2023-02-15 às 11 33 41" src="https://user-images.githubusercontent.com/8357343/219057403-54528818-3be7-452d-b62c-c6023e548c53.png">

Reply:
> > @ricardobrg what is this example/ stuff?
> 
> Pub is requiring an example app to improve our grade. I have added an empty example app, that will use the same code we have in the docs examples.

Okay, then shouldn't we add some example code instead of a blank project?

Reply:
> > > @ricardobrg what is this example/ stuff?
> > 
> > 
> > Pub is requiring an example app to improve our grade. I have added an empty example app, that will use the same code we have in the docs examples.
> 
> Okay, then shouldn't we add some example code instead of a blank project?

Yes, but I didn't want to add he code here since it was for docs. I have created another issue for that. #257